### PR TITLE
React queries

### PR DESCRIPTION
**Pull request by akakitani**
This handles loading real data from the db into the clientside stuff. I'll do submissions in another PR, that's also more work than I thought it would be so I wanna keep the PRs small

The maps don't look like they work with test data on the "deliver" page but this is just because the random longitude/latitude values are all over the world, with real locations they should be working fine

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -16,6 +16,8 @@ gem 'react-rails', '~> 0.11.1.0'
 gem 'underscore-rails'
 gem 'underscore-string-rails'
 gem 'uglifier', '>= 1.3.0'
+# Rails objects -> JS ovjects
```
Comment (jjwon) [inline Gemfile]: :s/ovjects/objects/g

File: app/assets/javascripts/react_components/coordinator/schedule.js.jsx [hunk 2]
```diff
@@ -138,6 +62,7 @@ var ScheduleDashboard = React.createClass({
     },
     render: function() {
         var content = _.isEmpty(this.state.donations) ? this.renderEmpty() : this.renderHasDonations();
+        console.log(this.state.recipients)
```
Comment (jjwon) [inline app/assets/javascripts/react_components/coordinator/schedule.js.jsx]: remove?

File: app/views/coordinator/deliver.html.erb [hunk 3]
```diff
@@ -3,6 +3,10 @@
     <div id="deliver-content"></div>
 </div>
 
+<script>
+console.log(gon.deliveries)
+</script>
```
Comment (jjwon) [inline app/views/coordinator/deliver.html.erb]: remove?

**Comment by jjwon** [comment]
just to be sure, this *doesn't* fix #24, right?

PR GAME 3 :100: letsgo

**Comment by akakitani** [comment]
nope i didn't touch anything for #24

**Pull request merged**